zulipterminal/ui_tools/views.py

```diff
@@ -293,13 +293,17 @@ def __init__(self) -> None:
 class StreamsView(urwid.Frame):
     def __init__(self, streams_btn_list: List[Any], view: Any) -> None:
         self.view = view
-        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
+        # Create Stream List Box
```

Comment:
While comments are a good thing to include, it is always recommended to write code in a way that is readable, self-explanatory and self-documenting instead.

 When the code already explains what is happening, it is superfluous to include an obvious comment. :)

Reply:
So, all the comments I have done are so that I can group related variables together. It would be confusing to know in a single glance what the variables are for and where they are used. Infact, even `empty_search` and `search_lock` can be brought above.
I think `Create` can be removed from the comments, so that the single line actual creation is not confused with.

Reply:
I've made some changes to this part in the first refactor commit. It might be useful doing this?

Reply:
Hmm. I am not entirely sure about the refactor. Perhaps @neiljp could pitch in.

Reply:
I appreciate the desire to tidy the code, but this might be reasonable as part of a larger cleanup like deduplicating the stream+topic list.